src/OSPSuite.Core/Domain/Builder/IndividualBuildingBlock.cs

```diff
@@ -1,7 +1,22 @@
-﻿namespace OSPSuite.Core.Domain.Builder
+﻿using OSPSuite.Core.Domain.Services;
+using OSPSuite.Utility.Extensions;
+
+namespace OSPSuite.Core.Domain.Builder
 {
    public class IndividualBuildingBlock : PathAndValueEntityBuildingBlockFromPKSim<IndividualParameter>
    {
       public OriginDataItems OriginData { get; } = new OriginDataItems();
+
+      public override void UpdatePropertiesFrom(IUpdatable source, ICloneManager cloneManager)
+      {
+         base.UpdatePropertiesFrom(source, cloneManager);
+         var sourceIndividualBuildingBlock = source as IndividualBuildingBlock;
+         if (sourceIndividualBuildingBlock == null)
+            return;
+         
+         OriginData.ValueOrigin.UpdateAllFrom(sourceIndividualBuildingBlock.OriginData.ValueOrigin);
+
+         sourceIndividualBuildingBlock.OriginData.AllDataItems.Each(x => OriginData.AddOriginDataItem(x));
```

Comment:
This could potentially be a method from your OriginDataClass (either CLone or UdateFrom) to keep with what we typically do. Otherwise, looks good